mods/ts/sequences/civilian.yaml

```diff
@@ -368,17 +389,22 @@ capyr03:
 ^city2x3:
 	Inherits: ^cabase
 	Defaults:
-		Offset: 12, -30, 30
+		Offset: 12, -30, 36
```

Comment:
The bottom-right edges were cut off (invisible) with the previous values. If there's a better way to fix that, I'm all ears/eyes.